packages/fx-core/src/plugins/resource/simpleauth/constants.ts

```diff
@@ -53,12 +50,17 @@ export class Constants {
     },
   };
 
-  static readonly ResourcesFolderName: string = path.join(
```

Comment:
Is this value not used and you just clean up the code? I searched the changes and seems this PR doesn't remove any code that uses this variable.

Reply:
Yes. This variable is not used so I clean up the code.